src/test/java/com/epam/tc/hw2/exercise1/SiteStructureTests.java

```diff
@@ -0,0 +1,242 @@
+            By.xpath("//header//ul[@class='uui-navigation nav navbar-nav m-l8']/li"));
+        softAssertions.assertThat(menuElements.size())
+                      .as("The menu in header section should have 4 top elements")
+                      .isEqualTo(4);
+
+        WebElement menuHome = webDriver.findElement(By.xpath("//header/.//a[contains(text(), 'Home')]"));
+        softAssertions.assertThat(menuHome.getText())
+                      .as("The menu element 'home' is displayed as 'HOME'")
+                      .isEqualTo("HOME");
+        softAssertions.assertThat(menuHome.isDisplayed())
+                      .as("The menu element 'HOME' should be visible").isTrue();
+
+        WebElement menuContactForm = webDriver.findElement(By.xpath("//header/.//a[contains(text(), 'Contact form')]"));
+        softAssertions.assertThat(menuContactForm.getText())
+                      .as("The menu element 'Contact form' is displayed as 'CONTACT FORM'")
+                      .isEqualTo("CONTACT FORM");
+        softAssertions.assertThat(menuContactForm.isDisplayed())
+                      .as("The menu element 'CONTACT FORM' should be visible").isTrue();
+
+        WebElement menuService = webDriver
+            .findElement(By.xpath("//header/.//a[@class='dropdown-toggle' and  @data-toggle='dropdown']"));
+        softAssertions.assertThat(menuService.getText())
+                      .as("The menu element 'Service' is displayed as 'SERVICE'")
+                      .isEqualTo("SERVICE");
+        softAssertions.assertThat(menuService.isDisplayed())
+                      .as("The menu element 'SERVICE' should be visible").isTrue();
+
+        WebElement menuMetalsAndColors = webDriver
+            .findElement(By.xpath("//header/.//a[contains(text(), 'Metals & Colors')]"));
+        softAssertions.assertThat(menuMetalsAndColors.getText())
+                      .as("The menu element 'Metals & Colors' is displayed as 'METALS & COLORS'")
+                      .isEqualTo("METALS & COLORS");
+        softAssertions.assertThat(menuMetalsAndColors.isDisplayed())
+                      .as("The menu element 'METALS & COLORS' should be visible").isTrue();
+
+        // 6. Assert that there are 4 images on the Index Page, and they are displayed
+        List<WebElement> imagesOnIndexPage =
+            webDriver.findElements(By.xpath("//span[contains(@class,'icons-benefit')]"));
+        softAssertions.assertThat(imagesOnIndexPage.size())
+                      .as(" There are should be 4 images on the Index Page")
+                      .isEqualTo(4);
+
+        for (WebElement imageElement : imagesOnIndexPage) {
+            softAssertions.assertThat(imageElement.isDisplayed())
+                          .as("The image element with class = '"
+                              + imageElement.getAttribute("class")
+                              + "' should be visible").isTrue();
+        }
```

Comment:
assertions inside loop is not a good practice, please use method `allMatch` or similar